### DISCUSSION TITLE
My node can't keep its peers

### DISCUSSION THREAD

**Discussion by Falci** [lightningnetwork/lnd#8982] [Q&A] [answered]
When I check ThunderHub, it says all my channels (4) are red/inactive. When I check `lnd` logs, among other things, I see:

```
CHRE: The chain backend has an insufficient number of connected outbound peers (0 connected, expected minimum is 6) which can be a security issue. Connect to more trusted nodes manually if necessary.
```

My setup:
- Raspberry Pi 5
- Ubuntu 24.04
- lnd version 0.18.2-beta commit=v0.18.2-beta
- Bitcoin Core version v27.1.0
- Tor version 0.4.8.10.

I opened my router port 9735 and mapped to the Pi's MAC, in case someone tries to connect via clearnet (was it needed?)

**Comment by guggero** [comment, marked as answer]
> `CHRE: The chain backend`

That's `bitcoind`. So check your config there and add static (trusted, if you know someone who also runs a node) peers and/or enable clearnet on `bitcoind` as well.
